src/join/composer.jl

```diff
@@ -640,13 +640,13 @@ change in future releases.
 - `df1`, `df2`, `dfs...`: the `AbstractDataFrames` to be joined
 
 # Keyword Arguments
-- `on` : A column name to join `df1` and `df2` on. If the columns on which
+- `on` : The key column name to join `df1` and `df2` on. If the columns on which
```

Comment:
while we are at it maybe correct also the fact that it can be column or columns?

Reply:
I've pushed this change for all docstrings (with a small edit, replacing `mix names and pairs` with `mix names and name pairs`).